auth/README.md

```diff
@@ -0,0 +1,98 @@
+# Generate custom JWT for service-to-service authentication
+There are two forms of JSON Web Token (JWT) signed by Google: A Google ID token or a custom JWT that 
+is signed only by the service account of the caller. The following two endpoint security definitions
+can be used:
+```
```

Comment:
Since you mentioned (1) Google ID token and (2) a custom JWT above, the order of the security definitions here needs to be switched (Google ID token be the first).